translate/cloud-client/beta_snippets.py

```diff
@@ -0,0 +1,350 @@
+    from google.cloud import translate_v3beta1 as translate
+    client = translate.TranslationServiceClient()
+
+    # project_id = YOUR_PROJECT_ID
+    location = 'global'
+
+    parent = client.location_path(project_id, location)
+
+    response = client.get_supported_languages(parent)
+
+    print('Supported Languages:')
+    for language in response.languages:
+        print('Language Code: {}'.format(language.language_code))
+    # [END translate_list_codes_beta]
+
+
+def list_languages_with_target(project_id, display_language_code):
+    # [START translate_list_language_names_beta]
+    from google.cloud import translate_v3beta1 as translate
+    client = translate.TranslationServiceClient()
+
+    # project_id = YOUR_PROJECT_ID
+    # display_language_code = 'is'
+    location = 'global'
+
+    parent = client.location_path(project_id, location)
+
+    response = client.get_supported_languages(
+        parent=parent,
+        display_language_code=display_language_code)
+
+    print('Supported Languages:')
+    for language in response.languages:
+        print('Language Code: {}'.format(language.language_code))
+        print('Display Name: {}\n'.format(language.display_name))
+    # [END translate_list_language_names_beta]
+
+
+def create_glossary(project_id):
+    # [START translate_create_glossary_beta]
+    from google.cloud import translate_v3beta1 as translate
+    client = translate.TranslationServiceClient()
+
+    # project_id = 'YOUR_PROJECT_ID'
+    location = 'us-central1'  # The location of the glossary
+
+    name = client.glossary_path(
+        project_id,
+        location,
+        'glossary')  # The name of your glossary
+
+    language_codes_set = translate.types.Glossary.LanguageCodesSet(
+        language_codes=['en', 'es'])
+
+    gcs_source = translate.types.GcsSource(
+        input_uri='gs://cloud-samples-data/translation/glossary.csv')
+
+    input_config = translate.types.GlossaryInputConfig(
+        gcs_source=gcs_source)
+
+    glossary = translate.types.Glossary(
+        name=name,
+        language_codes_set=language_codes_set,
+        input_config=input_config)
+
+    parent = client.location_path(project_id, location)
+
+    operation = client.create_glossary(parent=parent, glossary=glossary)
+
+    result = operation.result(timeout=90)
+    print('Created: {}'.format(result.name))
+    print('Input Uri: {}'.format(result.input_config.gcs_source.input_uri))
+    # [END translate_create_glossary_beta]
+
+
+def list_glossaries(project_id):
+    # [START translate_list_glossary_beta]
+    from google.cloud import translate_v3beta1 as translate
+    client = translate.TranslationServiceClient()
+
+    # project_id = 'YOUR_PROJECT_ID'
+    location = 'us-central1'  # The location of the glossary
+
+    parent = client.location_path(project_id, location)
+
+    for glossary in client.list_glossaries(parent):
+        print('Name: {}'.format(glossary.name))
+        print('Language Pair:')
```

Comment:
Ah, we changed this.

I will update this to list Language Codes Set (because that's what we create!)